apps/src/music/views/Callouts.tsx

```diff
@@ -76,6 +76,9 @@ const availableCallouts: AvailableCallouts = {
   'trigger-button-2': {selector: `#${Triggers[1].id}`},
   'trigger-button-3': {selector: `#${Triggers[2].id}`},
   'trigger-button-4': {selector: `#${Triggers[3].id}`},
+  'start-over-button': {
+    selector: 'button:has(i.fa-refresh)',
```

Comment:
It would probably be more robust to give the button an ID and then the selector can simply be something like `#start-over-button`, similar to what we do for the Run button.

Reply:
Good call. Updated.